src/data/sidebar.ts

```diff
@@ -84,6 +84,7 @@ export const sidebarContent: ISidebarContent = [
       makePage("Volumes", "reference"),
       makePage("Cron Jobs", "reference"),
       makePage("Pricing", "reference"),
+      makePage("Scale to Zero", "reference"),
```

Comment:
Thoughts on renaming this from "Scale to Zero" to "App Sleeping" entirely?

We use the term "App Sleeping" when exposing this feature, and "Scale to Zero" is more of a "marketing" artifact than a feature - I think we should stay consistent in naming features, and only use a single name everywhere except marketing content that tells a story about what the feature enables you to do

I think it might also be confusing reading this from top-down going from "scale to zero" to "app sleeping" because it leaves too much effort to the reader to figure out the correlation

Reply:
We talked about this, and you gave the vote we needed.  Changed to App Sleeping